### PR TITLE
Prince newbranch

### PR DESCRIPTION
**Pull request by princegeorge121**
Previous request was not able to merge. So created a new request

### CODE REVIEW

File: client/mycart/src/app/services/order.service.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  postOrders(model: OrderCreateDto) {
+    return this.http.post<CategoryViewDto>(this.url, model);
+  }
+
+  delete(id: number) {
+    return this.http.delete(`${this.url}/${id}`)
+  }
+
+
```
Comment (mishelshaji) [inline client/mycart/src/app/services/order.service.ts:25]: Remove the extra blank line.

File: server/MyCart/MyCart.Services/Dto/OrderCreateDto.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyCart.Services.Dto
+{
+    public class OrderCreateDto
+    {
+
```
Comment (mishelshaji) [inline server/MyCart/MyCart.Services/Dto/OrderCreateDto.cs:16]: Remove the blank line.

File: server/MyCart/MyCart.Services/Services/OrderServices.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            Order order = new()
+            {
+                CustomerId = dto.CustomerId,
+                DeliveryAddress = dto.DeliveryAddress,
+                OrderTime = dto.OrderTime,
+                PaymentStatus = dto.PaymentStatus,
+                TotalPrice = dto.TotalPrice
+            };
+
+
```
Comment (mishelshaji) [inline server/MyCart/MyCart.Services/Services/OrderServices.cs:55]: Remove the extra blank line.

File: server/MyCart/MyCart/Program.cs [hunk 4]
```diff
@@ -9,6 +9,7 @@
 builder.Services.AddSwaggerGen();
 builder.Services.AddScoped<CategoryServices>();
 builder.Services.AddScoped<ProductServices>();
+builder.Services.AddScoped<OrderServices>();
 builder.Services.AddScoped<CartServices>();
```
Comment (mishelshaji) [inline server/MyCart/MyCart/Program.cs:15]: Remove the extra blank line.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/MyCart/MyCart/Program.cs (hunk 4 above)
Comment (princegeorge121) [reply server/MyCart/MyCart/Program.cs:15]: Done.

File: server/MyCart/MyCart.Services/Services/OrderServices.cs (hunk 3 above)
Comment (princegeorge121) [reply server/MyCart/MyCart.Services/Services/OrderServices.cs:55]: Done.

File: client/mycart/src/app/services/order.service.ts (hunk 1 above)
Comment (princegeorge121) [reply client/mycart/src/app/services/order.service.ts:25]: Done.

File: server/MyCart/MyCart.Services/Dto/OrderCreateDto.cs (hunk 2 above)
Comment (princegeorge121) [reply server/MyCart/MyCart.Services/Dto/OrderCreateDto.cs:16]: Done.

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**